src/modules/ie-objects/ObjectDetailPage.tsx

```diff
@@ -1182,6 +1186,21 @@ export const ObjectDetailPage: FC<DefaultSeoInfo> = ({ title, description, image
 		iiifViewerReference?.current?.setActiveWordIndex(index);
 	};
 
+	const handleIiifViewerSelection = (rect: TextLine) => {
```

Comment:
SUGGESTION: This file is getting a bit too long, isn't it a good idea to begin splitting functions up in separate helper files?